### PR TITLE
Update for Python3-only since Python2 is EoL on 2019-12-31 and made a…

### PR DESCRIPTION
**Pull request by BasementTrix**
…nimated output flow better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,16 @@ LOLcat
 ======
 
 Straight Python port of the Ruby version at https://github.com/busyloop/lolcat/
-but with 100% more fun, because, hey, it’s not Ruby. There are no external dependencies..
+but with 100% more fun, because, hey, it’s not Ruby. There are no external dependencies...
 
 It's a tool like cat, with added lulz.
 
+Forked from https://github.com/tehmaze/lolcat
```
Comment (jmtd) [inline README.md]: I'm guessing this isn't meant to have made it to this PR…

File: README.md [hunk 2]
```diff
@@ -2,10 +2,16 @@ LOLcat
...
+
+Pulled Windows Support from https://github.com/naturalBornPixel/lolcat_windows
```
Comment (jmtd) [inline README.md]: That's #11 so maybe shouldn't be in this PR too?

**Comment by BasementTrix** [comment]
On Thu, May 09, 2019 at 02:58:50AM -0700, Jonathan Dowland wrote:
>  It's a tool like cat, with added lulz.
>  
> +Forked from https://github.com/tehmaze/lolcat
> 
> I'm guessing this isn't meant to have made it to this PR…

Probably not.  Still new to the whole github thing.  More accustomed to
e-mailing diff outputs.

-- 
John D. "Trix" Farrar            __\\|//__               Basement.NET
trix@basement.net                (` o-o ')   http://www.basement.net/
--------------------------------ooO-(_)-Ooo--------------------------
 GPG Key Fprint: 39D0 BCD2 9C9F B7EF B125  D419 61E8 8648 16F2 1814

**Comment by BasementTrix** [comment]
On Thu, May 09, 2019 at 02:59:43AM -0700, Jonathan Dowland wrote:
> +Pulled Windows Support from https://github.com/naturalBornPixel/lolcat_windows
> 
> That's #11 so maybe shouldn't be in this PR too?

The PR hsa been out there for quite some time.  Wasn't sure if it was
going to get integrated, so I pulled it into my repository and then,
on a lark, figured I'd send a PR back in case anyone was watching.

Apologies if I've created a mess for you. That wasn't my intent.

-- 
John D. "Trix" Farrar            __\\|//__               Basement.NET
trix@basement.net                (` o-o ')   http://www.basement.net/
--------------------------------ooO-(_)-Ooo--------------------------
 GPG Key Fprint: 39D0 BCD2 9C9F B7EF B125  D419 61E8 8648 16F2 1814

**Comment by jmtd** [comment]
> Apologies if I've created a mess for you. That wasn't my intent.

Not at all, I'm not even a maintainer, I just noticed your PR when I got a notification that my (trivial one line doc fix) PR was merged. I'm not sure why I fell into review mode, except I thought it might help increase the chances of this getting merged. But that's up to the maintainer so perhaps I should leave them to their jobs ;)

**Comment by BasementTrix** [comment]
Got too far out of sync as development continued on basementtrix/rcat (nee basementtrix/lolcat).

Perhaps a new PR later.

**Pull request closed without merging**